src/components/spatial/Spatial.js

```diff
@@ -211,9 +224,11 @@ export default function Spatial(props) {
     },
     visible: areCellsOn,
     ...cellLayerDefaultProps(cellsData, updateStatus, updateCellsHover, uuid),
+    getLineWidth: cellOpacity < 0.7 ? lineWidthScale : 0,
```

Comment:
Sorry, I meant the deck.gl prop called `lineWidthScale`, it is listed in the documentation, but I have not tried it myself https://deck.gl/docs/api-reference/layers/polygon-layer#linewidthscale although this should do the same thing